developer/src/common/web/utils/src/utils/KeymanSentry.ts

```diff
@@ -64,11 +64,17 @@ export class KeymanSentry {
     }
   }
 
-  static async captureException(e: any): Promise<never> {
+  /**
+   * capture an exception for Sentry; note that in local environments, this will normally
+   * throw the exception rather than sending to Sentry
+   * @param e
+   * @param force if true, reports to Sentry even in local environments
```

Comment:
Does `force` work as an option in `SentryNodeOptions`?

Or is it `TestKeymanSentry` the only intended usage?

Reply:
> Does `force` work as an option in `SentryNodeOptions`?

No.

> Or is it `TestKeymanSentry` the only intended usage?

Yes